docs/source/accelerators/mps_basic.rst

```diff
@@ -0,0 +1,41 @@
+:orphan:
+
+.. _mps_basic:
```

Comment:
Shall we include a warning that this is still experimental and further work within PyTorch is happening?

Reply:
yes we should! Good point